Sage/carbon
# chore(carousel): remove classic theme

removal of classic theme from the component and the classic theme example from storybook.

BREAKING CHANGE: The classic theme has been removed from carousel. To upgrade
please use a DLS theme.

BREAKING CHANGE: The `transition` prop has been removed from carousel. This was used in the classic theme.


fixes FE-2908

### Proposed behaviour
Removal of classic theme from the Carousel component and removal of the classic theme storybook example.

### Current behaviour
Component currently supports classic theme and classic theme examples in storybook.

### Checklist


<del>- [ ] Screenshots are included in the PR</del>
<del>- [ ] Carbon implementation and Design System documentation are congruent</del>
<del>- [ ] Cypress automation tests added or updated</del>

### Additional context
N/A

### Testing instructions
Pull down the branch.
Open storybook using `npm start`
Check the carousel component was as it previously did.
Check that the classic story for Carousel is not present.
